_episodes/73.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: episode
+title: "73: Wrapping Trino packages with a bow"
+date: 2025-04-09
+tags: trino rpm docker image container tarball plugin
+youtube_id: "7EyTcC3k-CY"
+wistia_id:
+sections:
+- time: 0:00
+  title: Introduction with Cole and Manfred
+- time: 1:49
+  title: Release 473 and 474
+- time: 7:17
+  title: Other news
+- time: 8:03
+  title: New Trino book
+- time: 11:08
+  title: Trino Query UI introduction
+- time: 16:55
+  title: Trino packaging history and overview
+- time: 22:15
+  title: Improvements project
+- time: 23:05
+  title: Plugin documentation and overview
+- time: 27:40
+  title: trino-packages repository and RPM module
+- time: 32:00
+  title: custom tarball module
+- time: 42:13
+  title: custom docker container 
+- time: 54:27
+  title: Upcoming events
+introduction: | 
+  Manfred Moser talks about his recent work to improve Trino packaging and demos
+  how to create your own RPM, tarball, and container image.
+---
+
+## Hosts
+
+* [Manfred Moser](https://www.linkedin.com/in/manfredmoser), Dev Rel Engineer
+  at [Chainguard](https://chainguard.dev)
+  [@simpligility](https://x.com/simpligility)
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Firebolt](https://www.firebolt.io/) 
+  
+## Releases
+
+Following are some highlights of the recent releases:
+
+[Trino 473]({{site.baseurl}}/docs/current/release/release-473.html)
+
+* Add support for array literals.
+* Add LDAP-based group provider.
+* Remove the deprecated glue-v1 metastore type.
+* Remove the deprecated Databricks Unity catalog integration.
+* Remove the Kudu connector.
+* Remove the Phoenix connector.
+
+But don't use 473 since there were [some breaking changes](https://github.com/trinodb/trino/issues/25381), fixed in...
+
+[Trino 474]({{site.baseurl}}/docs/current/release/release-474.html)
+
+* Fix a correctness bug in `GROUP BY` or `DISTINCT` queries with a large number
+  of unique groups.
+* Add `originalUser` and `authenticatedUser` as resource group selectors.
+* Use JDK 24 as the runtime in the Docker container. 
+
+As always, numerous performance improvements, bug fixes, and other features were
+added as well. Java 24 is coming as requirement soon - test the container!
+
+Releases continue to be slower. Trino needs your help.
```

Comment:
Do we want to expand on this? We didn't really emphasize it on the broadcast.

Reply:
No .. I will remove